### PR TITLE
Update phoebe.spec

### PR DESCRIPTION
**Pull request by mge1512**
phoebe.spec as tested in OpenBuildService.

### CODE REVIEW

File: dist/rpm/phoebe.spec [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+In many circumstances, operators are used to deal with telemetry, live charts, alerts, etc. which could help them identifying the offending machine(s) and (re)act to fix any potential issues.
+
+However, one question comes to mind: wouldn't it be awesome if the machine could auto-tune itself and provide a self-healing capability to the user? Well, if that is enough to trigger your interest then this is what Phoeβe aims to provide.
+
+Phoeβe uses system telemetry as the input to its brain and produces a big set of settings which get applied to the running system. The decision made by the brain is continuously reevaluated (considering the grace_period setting)
+to offer eventually the best possible setup.
+
+%prep
+%setup -q
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:49]:
```suggestion
%autosetup
```

File: dist/rpm/phoebe.spec [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+meson build
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:50]: Otherwise the correct optflags will not be set
```suggestion
%meson
```

File: dist/rpm/phoebe.spec [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+%build
+cd build
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:53]:
```suggestion
```

File: dist/rpm/phoebe.spec [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+meson compile
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:54]:
```suggestion
%meson_build
```

File: dist/rpm/phoebe.spec [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+mkdir -p /${RPM_BUILD_ROOT}/%{_sysconfdir}/phoebe
+mkdir -p /${RPM_BUILD_ROOT}/%{_datadir}/phoebe
+install -m 755 build/src/phoebe                 /${RPM_BUILD_ROOT}/%{_bindir}
+install -m 644 build/src/libnetwork_plugin.so   /${RPM_BUILD_ROOT}/%{_libdir}/phoebe
+install -m 644 settings.json                    /${RPM_BUILD_ROOT}/%{_sysconfdir}/phoebe/settings.json
+install -m 644 csv_files/rates.csv              /${RPM_BUILD_ROOT}/%{_datadir}/phoebe/rates.csv
+
+%files
+%license LICENSE
+%doc OWNERS README.md CONTRIBUTING.md SETUP.md
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:68]: `OWNERS` and `CONTRIBUTING.md` are not really useful for end user documentation.

```suggestion
%doc README.md SETUP.md
```

File: dist/rpm/phoebe.spec [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+Name:           phoebe
+Version:        0.0.4
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:19]:
```suggestion
Version:        0.1
```

File: dist/rpm/phoebe.spec [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+Release:        0.3
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:20]: At least for building on OBS, the release tag is ignored and set by the build system.
```suggestion
Release:        0
```

File: dist/rpm/phoebe.spec [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+Summary:        Phoeβe wants to add basic artificial intelligence capabilities to the Linux OS
+License:        BSD-3-Clause
+URL:            https://github.com/SUSE/phoebe
+Source:         phoebe-%version.tar.xz
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:24]: This way spectool can download the latest version directly:

```suggestion
Source:         %{URL}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

File: dist/rpm/phoebe.spec [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+%prep
+%setup -q
+meson build
+
+%build
+cd build
+meson compile
+
+%install
+mkdir -p /${RPM_BUILD_ROOT}/%{_bindir}
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:57]: This can all be replaced with `%meson_install`

File: dist/rpm/phoebe.spec [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+URL:            https://github.com/SUSE/phoebe
+Source:         phoebe-%version.tar.xz
+BuildRequires:  meson
+BuildRequires:  ninja
+BuildRequires:  python3-devel
+BuildRequires:  python3-cffi
+BuildRequires:  python3-pycparser
+BuildRequires:  libjson-c-devel
+BuildRequires:  libnl3-devel
+Group:          System/Benchmark
```
Comment (dcermak) [inline dist/rpm/phoebe.spec:32]: The Group tag is obsolete:
```suggestion
```

**Review by dcermak — CHANGES_REQUESTED** [review]
Thanks for the PR @mge1512! I've added a few suggestions how the spec can be simplified.

**Comment by mvarlese** [comment]
Thanks @mge1512 

@dcermak I like the idea to have the SPEC file merged as all the changes look more as nice-to-have than roadblocks. In that way, we could refine the SPEC file later. Also consider the SPEC file is working just fine as a package is already being built in OBS.

**Pull request merged**